src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy

```diff
@@ -129,7 +129,7 @@ class DexMethodCountTask extends DefaultTask {
         printToFile(outputFile) { PrintStream out ->
             print(tree, out)
         }
-        outputTime = System.currentTimeMillis()
+        outputTime = java.lang.System.currentTimeMillis()
```

Comment:
Because of the inclusion of the System interface from the SDK Manager files.  Conflicts now because currentTimeMillis() isnt' in that interface.  I could have added that to that interface and made it a static on Real and changed this to System.Real.currentTimeMillis() but that didn't seem to buy anything.  Other alternative is I can rename the System file from the SDK Manager.